src/components/Info.js

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
```

Comment:
Why `"` instead of `'`?

Reply:
I really don't care as long as it's always the same. The default seems to be `"`. Do you have any preference?